javascript/javascript1/week4/VoiceAssistant.js

```diff
@@ -0,0 +1,72 @@
+const myObj = {
+    name: 0,
+    toDo: []
+}
+function getReply(command) {
+    if (command.split(' ').shift() === 'Set') {
+        let time = command.split('Set a timer for ').pop().split(' minutes')[0];
+        console.log(`Timer set for ${time} minutes`);
+        setTimeout(message, (time*60)*1000);       function message() {
+            console.log('Timer done');
+        }
+    }
+    else if (command.split(' ').shift() === 'Hello') {
+        myObj.name = command.split(' ').pop();
```

Comment:
I would recommend you to have string like "Hello, my name is", so your command is more precise ☺️